src/Button/types.ts

```diff
@@ -8,7 +8,7 @@ export const StyledButton = styled.button<SxProp>`
   ${sx};
 `
 
-export type VariantType = 'default' | 'primary' | 'invisible' | 'danger' | 'outline'
+export type VariantType = 'default' | 'primary' | 'invisible' | 'danger' | 'outline' | 'link'
```

Comment:
Is there any visual difference between `'invisible'` and `'link'`?

Reply:
@mperrotti yes, it has so height/padding and "link like" hover styles (underline). Its a button pretending to be a link, while `invisible` has buttony sizing, hover states, and is sometimes a muted color.